src/argilla/server/contexts/datasets.py

```diff
@@ -515,3 +515,21 @@ async def upsert_suggestion(
         schema=SuggestionCreateWithRecordId(record_id=record.id, **suggestion_create.dict()),
         constraints=[Suggestion.record_id, Suggestion.question_id],
     )
+
+
+async def delete_suggestions(db: "AsyncSession", record: Record, suggestions_ids: List[UUID]) -> None:
```

Comment:
Should we rename this to keep consistency with the rest of the function naming?

```suggestion
async def delete_suggestions_by_ids(db: "AsyncSession", record: Record, suggestions_ids: List[UUID]) -> None:
```

Reply:
I would say to keep it like this. In the `get` something functions we tend to suffix them using the `by_<column_for_filter_condition>`, but for the deletions we always pass the IDs. We also have `delete_records`.